acq4/devices/Device.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, deviceManager: acq4.Manager.Manager, config: dict, name: str)
         self._lock_tb_ = None
         self.dm = deviceManager
         self.dm.declareInterface(name, ['device'], self)
+        self._config = config
```

Comment:
I feel like I've been told not to do this, that it's "here be dragons" territory, but it behaves well for all the hardware we've tested it on, so... maybe we call it `_untrrustworthyConfig` or some such, so we won't feel as bad later when it somehow fails?

Reply:
searching around, this line is also present in Microscope, Filter, ThorLab's Filter, and VimbaXCamera (I added that). there are lots of weird `self.config =` choices to be wary of, though. maybe that's all I was warned off of? okay, but then that makes this a shitty proliferation of duplicate ways to access the config.

Reply:
oh, we also have `self.__config =` in Optomech.